app/views/citizens/own_homes/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @form, url: form_path, method: :patch, local: true) do |form| %>
+
+      <%
+        options = [
+          { value: :mortgage,       label: action_t('.own_home.mortgage') },
+          { value: :owned_outright, label: action_t('.own_home.owned_outright') },
+          { value: :no,             label: action_t('.own_home.no') }
+        ]
+      %>
+
+      <%= form.govuk_collection_radio_buttons(:own_home, options, :value, :label, title: content_for(:page_title)) %>
+
+      <div class="govuk-!-padding-bottom-6"></div>
```

Comment:
can you please move this form to shared/forms/_own_homes_form?

Reply:
Done